src/index.js

```diff
@@ -130,9 +132,10 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
           events = [events];
         }
         events.forEach(eventName => {
-          const handlerOptions = !this.props.preventDefault && ['touchstart', 'touchmove'].indexOf(eventName) !== -1
-            ? { passive: true }
-            : null;
+          const handlerOptions =
```

Comment:
Then the solution, honestly, is "let's stop using Phantom because it's not compatible with the modern web". I know firefox recently added a headless feature, perhaps we can use that.